src/backend/drivers/cswordbiblemoduleinfo.cpp

```diff
@@ -74,29 +74,14 @@ QStringList *CSwordBibleModuleInfo::books() const {
     if (!m_bookList) {
         m_bookList = new QStringList();
 
-        // Initialize m_hasOT and m_hasNT
         if (!m_boundsInitialized)
             initBounds();
 
-        int min = 1; // 1 = OT
-        int max = 2; // 2 = NT
-
-        if (!m_hasOT)
-            min++; // min == 2
-
-        if (!m_hasNT)
-            max--; // max == 1
-
-        if (min > max) {
-            qWarning("CSwordBibleModuleInfo (%s) no OT and not NT! Check your config!", module().getName());
-        } else {
-            std::unique_ptr<sword::VerseKey> key(
-                    static_cast<sword::VerseKey *>(module().createKey()));
-            key->setPosition(sword::TOP);
-
-            for (key->setTestament(min); !key->popError() && key->getTestament() <= max; key->setBook(key->getBook() + 1)) {
-                m_bookList->append( QString::fromUtf8(key->getBookName()) );
-            }
+        for (sword::VerseKey key(m_lowerBound);
+             !key.popError() && key.compare(m_upperBound) <= 0;
+             key.setBook(key.getBook() + 1))
+        {
+            m_bookList->append( QString::fromUtf8(key.getBookName()) );
```

Comment:
The lazy and proud person (me) who has not yet written a code style guide and therefore has no valid reason to complain, still wants to point out that when he would have made the change, he would have removed the spaces from inside the parenthesis on this line. I apologize.

Reply:
))